x-pack/plugins/apm/e2e/cypress/support/step_definitions/apm.ts

```diff
@@ -7,14 +7,17 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
 import { loginAndWaitForPage } from '../../integration/helpers';
 
+/** The default time in ms to wait for a Cypress command to complete */
+export const DEFAULT_TIMEOUT = 60 * 1000;
+
 Given(`a user browses the APM UI application`, () => {
   // open service overview page
   loginAndWaitForPage(`/app/apm#/services`);
 });
 
 When(`the user inspects the opbeans-node service`, () => {
   // click opbeans-node service
-  cy.get(':contains(opbeans-node)').last().click({ force: true });
+  cy.get(':contains(opbeans-node)', { timeout: DEFAULT_TIMEOUT }).last().click({ force: true });
```

Comment:
Does this time out, or why is it necessary to increase the timeout?

Reply:
It was an initial attempt to fix the issue when the kibana is not up and running, since the cypress tests used to report an error regarding clicking on the opbeans-node. I'll revert this particular change as I don't think it's necessary.

Reply:
It's being ages since I added this, so it's required otherwise it won't work in the CI:
- https://apm-ci.elastic.co/job/apm-ui/job/apm-ui-e2e-tests-mbp/job/PR-61803/113/artifact/cypress/videos/apm.feature.mp4

I've just reverted my reverted commit to validate the above assumption.